### PR TITLE
Example scenario - SQL DW v2

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/data/data-warehouse.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+We have provided three sample cost profiles based on amount of traffic you expect to get:
+
+* [Small][small-pricing]: this correlates to a small data pipeline with part-time availability.
+* [Medium][medium-pricing]: this correlates to a midsize data pipeline with full availabilty.
+* [Large][large-pricing]: this correlates to a large data pipeline with full availability.
+
+## Related Resources
+
+This example scenario is based on a version of this architecture used by  [Maritz Motivation Solutions](https://maritz.com) For more information, see their [customer story][source-document].
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: How about moving this to a new section called "Next steps" or highlighting that if you want to start implementing it this is where you need to go.

File: docs/example-scenario/data/data-warehouse.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+### Security
+
+The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+For general guidance on designing secure solutions, see the [Azure Security Documentation][security].
+
+### Resiliency
+
+For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: If we don't have anything specific to say about resiliency, I think we should pull it out.  Eventually I'd like to talk about where we're using managed services, where single points of failure are, and what you need to think about to keep this up and running.

File: docs/example-scenario/data/data-warehouse.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+* [Azure Active Directory (Azure AD)](/azure/active-directory/) authenticates users who connect to the Analysis Services server through Power BI. Data Factory can also use Azure AD to authenticate to SQL Data Warehouse via a service principal or Managed Service Identity (MSI).
+
+### Alternatives
+
+* This example demonstrates using an on-premises SQL Server database and an external dataset as the data sources to integrate via the pipeline. However, this architecture is suitable for loading a wide range of both relational and non-relational data sources.
+* Data Factory is designed for automating your data pipeline workflows. For jobs requiring high performance, considering [using Data Factory with Polybase](/azure/data-factory/connector-azure-sql-data-warehouse#use-polybase-to-load-data-into-azure-sql-data-warehouse). For one-time or on-demand jobs, you could also use available tools like SQL Server bulk copy (bcp) and AzCopy. For a general comparison, see [Choosing a data pipeline orchestration technology in Azure](/azure/architecture/data-guide/technology-choices/pipeline-orchestration-data-movement).
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: What does "high performance" mean in this context?

File: docs/example-scenario/data/data-warehouse.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+* For comparisons of different relevant technology options, see the following:
+
+    * [Choosing a data pipeline orchestration technology in Azure](/azure/architecture/data-guide/technology-choices/pipeline-orchestration-data-movement)
+    * [Choosing a batch processing technology in Azure](/azure/architecture/data-guide/technology-choices/batch-processing)
+    * [Choosing an analytical data store in Azure](/azure/architecture/data-guide/technology-choices/analytical-data-stores)
+    * [Choosing a data analytics technology in Azure](/azure/architecture/data-guide/technology-choices/analysis-visualizations-reporting)
+
+## Considerations
+
+### Availability
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: This section feels a little bit like we're ticking boxes to say "yep, we talk about availability".  Let's go into a bit more detail or maybe even condense availability & durability into a single section.

File: docs/example-scenario/data/data-warehouse.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+* Consolidating data from numerous heterogenous data sources to a cloud-scale data platform
+* Transforming source data into a common taxonomy and structure for consistency and comparability.   
+* Leveraging highly parallelized data loading to support thousands of incentive programs without the expense of on-premises infrastructure deployment and maintenance
+* Significantly improving the ratio of time spent gathering and transforming data vs. time spent analyzing data
+
+This example scenario is relevant to organizations that want to an integrated data pipeline from multiple data sources into a cloud-scale platform for advanced analysis and business intelligence capabilities. Potential applications include sales and marketing, e-commerce, healthcare, government, or other industry solutions involving large volumes of data that can deliver high business value through analysis.
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Moved.

File: docs/example-scenario/data/data-warehouse.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+* Transforming source data into a common taxonomy and structure for consistency and comparability.   
+* Leveraging highly parallelized data loading to support thousands of incentive programs without the expense of on-premises infrastructure deployment and maintenance
+* Significantly improving the ratio of time spent gathering and transforming data vs. time spent analyzing data
+
+This example scenario is relevant to organizations that want to an integrated data pipeline from multiple data sources into a cloud-scale platform for advanced analysis and business intelligence capabilities. Potential applications include sales and marketing, e-commerce, healthcare, government, or other industry solutions involving large volumes of data that can deliver high business value through analysis. 
+
+Using managed Azure services such as Data Factory and SQL Data Warehousecan significantly reduce costs by leveraging Microsoft's expertise in globally distributed cloud-scale data storage and analysis. If you have additional data service needs, you should review the list of available [fully managed intelligent database services in Azure][product-category].
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: This one is aligned with the language of some of the other scenarios. We should revise the language and apply it across scenarios.

File: docs/example-scenario/data/data-warehouse.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+## Potential use cases
+
+Consider this solution for the following use cases:
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Agree, let's apply this more broadly too.

File: docs/example-scenario/data/data-warehouse.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+* Creation and incremental revision of a data warehouse that serves as a single source of truth for your organization's data.
```
[the remark this answers is not in the document]
Comment (MikeWasson) [reply docs/example-scenario/data/data-warehouse.md]: Term I've seen is "incremental loading"

[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: It's referring to incremental updates, but the parallelism options weren't great.
- "Creation of and incremental updates to..."
- " Creation and incremental updating of..."

I think I'll change it to "Establishing a data warehouse that..."

File: docs/example-scenario/data/data-warehouse.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+* Integration of relational data with binary data or other non-relational datasets.
+* Analysis of large datasets by business data analysts through a well-defined semantic model and robust visualization tools.
+
+## Architecture
+
+![Architecture for a data warehouse scenario in Azure][architecture]
+
+This example scenario is based on the [Azure automated enterprise BI reference architecture](/azure/architecture/reference-architectures/data/enterprise-bi-adf). The data flows through the solution as follows:
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Agree.

[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: At some point we need to discuss a broader vision for how we best facilitate "clicking through" these various artifacts.

File: docs/example-scenario/data/data-warehouse.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+1. Data from each data source is periodically exported into Azure Blob storage, which serves as a staging area for the source data. Each export contains only the data that has changed since the previous export. 
+2. Data Factory incrementally loads the staged data from Blob storage into staging tables in SQL Data Warehouse. Data cleansing and transformation happens at this stage, and Polybase can be used to parallelized the process for large volumes of data.
+3. After a new batch of data is loaded into the warehouse, a previously created Analysis Services tabular model is refreshed. This provides a semantic model that simplifies the analysis of business data and relationships by business analysts.
+4. Business analysts use Power BI to analyze data in the warehouse via the Analysis Services semantic model.
+
+### Components
```
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: I'll think about this one. For the data sources in this case study, we don't really have a defined rationale for its use beyond it being a repository of data that we want to include in the data warehouse.

File: docs/example-scenario/data/data-warehouse.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: If we add a new section for this, we should standardize that across other relevant scenarios.

File: docs/example-scenario/data/data-warehouse.md (hunk 3 above)
Comment (MikeWasson) [reply docs/example-scenario/data/data-warehouse.md]: My understanding is that ADF uses PolyBase. We shouldn't imply that ADF is not high performance. - it's a pipeline orchestrator, it doesn't do the actual work.

AzCopy and PolyBase are for two different things. Use AzCopy to move data from on-prem into Storage, use PolyBase to load from Storage into SQL DW

We don't recommend bcp for large data transfers.

File: docs/example-scenario/data/data-warehouse.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: I wonder if we shouldn't even include line items for the data source components, because it's really just illustrating several options out of many data sources that a customer might have.

File: docs/example-scenario/data/data-warehouse.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (MikeWasson) [reply docs/example-scenario/data/data-warehouse.md]: My $0.02: "Consider this for..." is more concise/direct than "This solution could also be helpful for..."

File: docs/example-scenario/data/data-warehouse.md (hunk 4 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Let's discuss this section more broadly next week.

File: docs/example-scenario/data/data-warehouse.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
+---
+title: Example scenario for data warehousing and analytics in Azure
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 08/16/2018
+---
+
+# Example scenario for data warehousing and analytics in Azure
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: Let's update the title to highlight the business problem.  How about something like "Building an enterprise data warehouse combining multiple data sources"

File: docs/example-scenario/data/data-warehouse.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+
+Build a data pipeline to integrate large amounts of data from multiple sources into a cloud-scale analytics platform. This scenario is relevant for:
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: I like the idea, but let's make it more of a conversation since it's the first thing you're reading.  My suggestion would be to reword it to:

This example scenario builds a data pipeline to integrate large amounts of data from multiple sources into a single cloud-scale analytics platform.  This specific scenario was modeled from a sales and marketing company, but the design patterns would be similar for many industries requiring advanced analytics of large datasets including e-commerce, retail, and healthcare.

File: docs/example-scenario/data/data-warehouse.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+* Sales and marketing
+* E-commerce and retail
+* Healthcare
+* Other solutions requiring advanced analysis of large datasets
+
+## The scenario
+
+A sales and marketing company builds incentive programs. These programs reward customers, suppliers, salespeople, and employees. Data is fundamental to these programs, and the company wants to improve the insights gained through data analytics using Azure.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: In this example, we're demonstrating a sales and marketing company that builds incentive programs.  These programs...

File: docs/example-scenario/data/data-warehouse.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+The company needs a modern approach to analysis data, so that decisions are made using the right data at the right time. The company's goals include:
+* Consolidating different kinds of data sources into a cloud-scale platform.
+* Transforming source data into a common taxonomy and structure, to make the data consistent and easily compared.
+* Loading data using a highly parallelized approach that can support thousands of incentive programs, without the high costs of deploying and maintaining on-premises infrastructure.
+* Greatly reducing the time needed to gather and transform data, so you can focus on analyzing the data.
+
+## Potential use cases
+
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: "In addition to the other industries we've mentioned above, the design patterns you see demonstrated here could be used for any customer who's looking to do the following"

File: docs/example-scenario/data/data-warehouse.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+* An on-premises [SQL Server Parallel Data Warehouse](/sql/analytics-platform-system) appliance can also be used for big data processing. However, operating costs are often much lower with a managed cloud-based solution like SQL Data Warehouse. 
+* For comparisons of other alternatives, see:
+    * [Choosing a data pipeline orchestration technology in Azure](/azure/architecture/data-guide/technology-choices/pipeline-orchestration-data-movement)
+    * [Choosing a batch processing technology in Azure](/azure/architecture/data-guide/technology-choices/batch-processing)
+    * [Choosing an analytical data store in Azure](/azure/architecture/data-guide/technology-choices/analytical-data-stores)
+    * [Choosing a data analytics technology in Azure](/azure/architecture/data-guide/technology-choices/analysis-visualizations-reporting)
+
+## Considerations
+
+The technologies chosen meet the company's requirements for scalability and availability, while helping them control costs.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: The technologies chosen in the architecture were picked because they met the company's requirements...

File: docs/example-scenario/data/data-warehouse.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+* The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+## Pricing
+
+Review a [pricing sample for a data warehousing scenario][calculator] via the Azure pricing calculator. Adjust the values to see how your requirements affect your costs.
+
+* [SQL Data Warehouse](https://azure.microsoft.com/pricing/details/sql-data-warehouse/gen2/) allows you to scale your compute and storage levels independently. Compute resources are charged per hour, and you can scale or pause these resources on demand. Storage resources are billed per terabyte.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: "Storage resources are billed per terabyte, so your costs will grow as you ingest more data."

File: docs/example-scenario/data/data-warehouse.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+* [Data Factory](https://azure.microsoft.com/pricing/details/data-factory/) costs are based on the number of read/write operations, monitoring operations, and orchestration activities performed in a workload.
+* [Analysis Services](https://azure.microsoft.com/pricing/details/analysis-services/) is available in developer, basic, and standard tiers. Instances are priced based on query processing units (QPUs) and available memory.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: Instances are priced based on query processing units (QPUs) and available memory.  So to keep costs down, minimize the number of queries you run, how much data they scan, and how often they run.

File: docs/example-scenario/data/data-warehouse.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+* The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+## Pricing
+
+Review a [pricing sample for a data warehousing scenario][calculator] via the Azure pricing calculator. Adjust the values to see how your requirements affect your costs.
+
+* [SQL Data Warehouse](https://azure.microsoft.com/pricing/details/sql-data-warehouse/gen2/) allows you to scale your compute and storage levels independently. Compute resources are charged per hour, and you can scale or pause these resources on demand. Storage resources are billed per terabyte.
+* [Data Factory](https://azure.microsoft.com/pricing/details/data-factory/) costs are based on the number of read/write operations, monitoring operations, and orchestration activities performed in a workload.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: So your data factory billing will increase with each additional data stream as well as how much data is being ingested by it.

File: docs/example-scenario/data/data-warehouse.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+* Azure Analysis Services can be [scaled out](/azure/analysis-services/analysis-services-scale-out) to reduce response times during high query workloads. You can also separate processing from the query pool, so that client queries aren't slowed down by processing operations. 
+* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+* The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+## Pricing
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: I really like the direction of this, but let's go into a bit more of the specifics of how costs are effected by design.  I've added a few things below.

File: docs/example-scenario/data/data-warehouse.md (hunk 14 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: In the spirit of minimizing boilerplate language and increasing the "skimmability" of these documents, should we shorten or eliminate this?  Will we end up with more repetition across these documents?

Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Will change to: "This example may also be relevant to the following:"

File: docs/example-scenario/data/data-warehouse.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
+---
+title: Building an enterprise data warehouse from multiple data sources
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: Given the feedback during our meeting, how about: "Data warehousing and analytics for sales and marketing" ?

File: docs/example-scenario/data/data-warehouse.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+* [Analysis Services](https://azure.microsoft.com/pricing/details/analysis-services/) is available in developer, basic, and standard tiers. Instances are priced based on query processing units (QPUs) and available memory. To keep your costs lower, minimize the number of queries you run, how much data they process, and how often they run.
+* [Power BI](https://powerbi.microsoft.com/pricing/) has different product options for different requirements. [Power BI Embedded](https://azure.microsoft.com/pricing/details/power-bi-embedded/) provides an Azure-based option for embedding Power BI functionality inside your applications. A Power BI Embedded instance is included in the pricing sample above.  
+
+## Next Steps
+
+* Review the [Azure reference architecture for automated enterprise BI](/azure/architecture/reference-architectures/data/enterprise-bi-adf), which includes instructions for deploying an instance of this architecture in Azure.
+* Read the [Maritz Motivation Solutions customer story][source-document]. This example scenario is based on a version of their architecture.
```
Comment (adamboeglin) [inline docs/example-scenario/data/data-warehouse.md]: How about: "Read the Maritz Motivation Solutions customer story.  That user story follows a similar design pattern for managing their customer data."

File: docs/example-scenario/data/data-warehouse.md (hunk 20 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Agree

File: docs/example-scenario/data/data-warehouse.md (hunk 21 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/data-warehouse.md]: Changed, with second sentence as: "That story describes a similar approach to managing customer data."

**Comment by alexbuckgit** [comment]
Closing PR in favor of new PR targeting master.

File: docs/example-scenario/data/data-warehouse.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+* Azure Analysis Services can be [scaled out](/azure/analysis-services/analysis-services-scale-out) to reduce response times during high query workloads. You can also separate processing from the query pool, so that client queries aren't slowed down by processing operations. 
+* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+* The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+## Pricing
```
Comment (rondunn) [inline docs/example-scenario/data/data-warehouse.md]: The article is missing the data volumes and load frequencies that would help to make sense of the pricing example. The numbers look random ... just vague increments of powers of 10 ... and there's nothing to relate them back to the data sources, data warehouse, etc. I'm calling this out because pricing of ADF V2 is a major customer challenge, and having an E2E worked example with traceable numbers / estimates would be helpful.
